www/docs/errors/no-main.md

```diff
@@ -0,0 +1,25 @@
+# Build does not contain a main function
+
+This usually happens if you're trying to build a library or if you didn't setup the `builds.main` section in your `.goreleaser.yml` and you `main.go` is not in the root folder.
```

Comment:
@caarlos0 , it is worth adding the error message. May improve SEO.

```
build failed after 0.11s error=build for ovn_exporter does not contain a main function
```

Reply:
good catch ,thanks! 🙏